check_disk_btrfs

```diff
@@ -162,12 +162,23 @@ def main():
     arg_parser.add_argument('-c', '--threshold-critical', type=int, default=90, help="critical threshold in percent")
     arg_parser.add_argument('-V', '--volume', default='/', help="btrfs volume") #TODO support multiple volumes
     arg_parser.add_argument("-v", "--verbose", action="store_true", help="increase output verbosity")
+    arg_parser.add_argument("--btrfs_path", default=None, help="Specify the btrfs path to the executable")
+    arg_parser.add_argument("--sudo_path", default=None, help="Specify the sudo path to the executable")
```

Comment:
I think it would be better to use the defaults here. 

Currently we have two global variables for the paths, you can remove those and add them as defaults in the argument. Then the if args.sudo_path checks can be removed as well.

Reply:
I made something ... I don't know if it's better or not.
To avoid consequent refacto, i keeped the global vars.
Using the default values in argparse allows to drop the if condition.
I set default values to get from environment then an hardcoded string, i wanted to keep the ability to pass via envvar.